packages/esm-patient-labs-app/src/config-schema.ts

```diff
@@ -41,16 +41,53 @@ export const configSchema = {
       _default: '52a447d3-a64a-11e3-9aeb-50e549534c5e',
     },
   },
+  labTests: {
+    _type: Type.Array,
+    _elements: {
+      labTestUuid: {
+        _type: Type.UUID,
+        _description: "UUID for the 'Lab' Test that require order reason",
+        _default: '',
+      },
+      labTestOrderReasons: {
+        _type: Type.Array,
+        _elements: {
+          uuid: {
+            _type: Type.UUID,
+            _description: 'Lab test concept UUID',
```

Comment:
Could we get a better description here? It's really unclear how this differs from `labTestUuid` just from the text and that description should be clear.